misc/install-deps.sh

```diff
@@ -21,23 +21,28 @@ case $distro in
     "ubuntu" | "debian" | "raspbian" | "kali" | "linuxmint")
         apt-get $OPT install pandoc libdw-dev python3-dev libncursesw5-dev pkg-config
         apt-get $OPT install libluajit-5.1-dev || true
+        [ $DEBUG ] && apt-get $OPT install libunwind-dev || true
         apt-get $OPT install libcapstone-dev || true ;;
-    "fedora")
+    "fedora"*)
```

Comment:
No wildcard plesae.  I'd like to fix it with ID_LIKE as in #1468.

Reply:
Thanks you for code review. :)